packages/chains/package.json

```diff
@@ -31,7 +31,12 @@
   "scripts": {
     "dev": "tsc --watch",
     "build": "tsup",
-    "clean": "rm -rf .turbo node_modules dist"
+    "clean": "rm -rf .turbo node_modules dist",
+    "lint": "eslint . --max-warnings 0",
+    "docs:check": "typedoc --skipErrorChecking --plugin typedoc-plugin-markdown --plugin typedoc-plugin-coverage",
+    "docs:build": "typedoc --skipErrorChecking --plugin typedoc-plugin-markdown --plugin typedoc-plugin-coverage",
+    "docs:watch": "typedoc --skipErrorChecking --plugin typedoc-plugin-markdown --plugin typedoc-plugin-coverage --watch",
+    "docs:serve": "typedoc --skipErrorChecking --plugin typedoc-plugin-markdown --plugin typedoc-plugin-coverage --watch --serve"
```

Comment:
Looks like some of the flags passes to `typedoc` are redundant to what's in the `typedoc.json` config file. Shouldn't `tsdoc` run here pick up the settings in the config file?

Reply:
good catch. should be fixed